Formula/carla.rb

```diff
@@ -0,0 +1,27 @@
+class Carla < Formula
+  desc "Audio plugin host supporting LADSPA, LV2, VST2/3, SF2 and more"
+  homepage "https://kxstudio.linuxaudio.org/Applications:Carla"
+  url "https://github.com/falkTX/carla", :using => :git, :tag => "v1.9.9", :revision => "c03571a9ef95ac0e9564b95347f5de819aa7fb54"
+  revision 0
+
+  depends_on "pkg-config" => :build
+  depends_on "fluid-synth"
+  depends_on "liblo"
+  depends_on "libmagic"
+  depends_on :macos => :mavericks
```

Comment:
Why?

Reply:
He's [patched this in master](https://github.com/falkTX/Carla/commit/2355f147b38c1d6647f413f1785dcc8be37eff41), but not in the stable.  Should I include the patch now and remove the compat flag so that the next version works without formula changes?  Please advise.

The update will look something like this:

```ruby
  def install
    args = []

    # Fix C++11 compilation
    if :macos < :mavericks?
      args << "MACOS_OLD=true"
    end

    system "make", *args
    system "make", "install", "PREFIX=#{prefix}"
  end
```

Reply:
Per 35a2e74, I've gone ahead and removed this restriction in favor of the `MACOS_OLD` flag.  The developer says this flag will remain for the foreseeable future.